docs/Data_Portal/Users_Guide/oncomatrix.md

```diff
@@ -301,7 +397,7 @@ Click `Submit` and the matrix will update to reflect the selected MSigDB gene se
 
 ## Variables
 
-The third button from the left called ``Variables` allows user to add in additional variables in the form of rows on the matrix. Click `Variables` to display a tree of variables and select `Disease type`, `Index date` and `Primary site`. Click the button `Submit 3 variables` as shown.
+The third button from the left called `Variables` allows user to add in additional variables in the form of rows on the matrix. Click `Variables` to display a tree of variables and select `Disease type`, `Index date` and `Primary site`. Click the button `Submit 3 variables` as shown.
 
 [<img src="./images/oncomatrix/46-variables.png" width="300"/>](./images/oncomatrix/46-variables.png 'Click to see the full image.')
```

Comment:
@siosonel Here's an example of an image that won't render.  I'll tag you in one that renders correctly soon.  It has something to do with being part HTML and part markdown

Reply:
Yes, the html syntax seems to have something to do with this, but it's weird that it works for others but not all. And if I replace the image filename that doesn't work with a filename that works, then the image loads, so it's more than a syntax issue. Sorry it's taking a bit to figure out, it's puzzling.